packages/mobile/src/analytics/ValoraAnalytics.ts

```diff
@@ -72,7 +72,7 @@ class ValoraAnalytics {
         this.deviceInfo = deviceInfo
         this.sessionId = Web3Utils.soliditySha3({
```

Comment:
Throughout the codebase we tend to use `sha256` from ethereumjs-util when we just need a generic hash function 
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/transactions/actions.ts#L74
and in `authentication.ts` and the web3 soliditySha3 for the verification hash identifier